libnmstate/nm/ipv4.py

```diff
@@ -119,3 +121,17 @@ def get_ip_profile(active_connection):
     if remote_conn:
         return remote_conn.get_setting_ip4_config()
     return None
+
+
+def get_route_running():
+    client = nmclient.client()
+    skip_networks = [iplib.IPV4_LINK_LOCAL_NETWORK]
```

Comment:
As we now split the config and running, there is no need to hide any routes.

Reply:
Done.